### PR TITLE
Docs consolidation #trivial

### PR DESCRIPTION
**Pull request by ashfurrow**
This moves the `emission/docs/` documentation to `docs/`, as well as cleanup the existing docs a bit. I moved Emission's old changelog into the `docs/` folder too. Many docs still point to the Emission repo, but they all use commit hashes in the URL, so they'll continue to work. It would be tedious to update them to point to Eigen, so I'm skipping that for now.

I also updated the `thanks.md` file, thanking significant contributors to the app.

### CODE REVIEW

File: docs/map_to_the_app.md [hunk 1]
```diff
@@ -72,4 +71,6 @@ There is [extensive inline documentation](https://github.com/artsy/emission/blob
 
 ## Formerly Preferred Practices
 
-These are techniques or approaches that we _used_ to use, but which have been replaced with newer concepts above. It's an empty list so far, but will grow with time.
+These are techniques or approaches that we _used_ to use, but which have been replaced with newer concepts above. This will grow with time.
+
+- We used to have a separate repo for Emission and Eigen, but [they were merged here](https://github.com/artsy/eigen/pull/3022).
```
Comment (brainbicycle) [inline docs/map_to_the_app.md]: 🎉

File: docs/thanks.md [hunk 2]
```diff
@@ -9,3 +9,13 @@ Thanks to all the contributors with 100+ commits from Eigen, and pre-OSS Eigen:
...
+- Kieran Gillen - [kierangillen](https://github.com/kierangillen) - [@kierangillen](https://twitter.com/kierangillen)
+- Maxim Cramer - [mennenia](https://github.com/mennenia) - [@mennenia](https://twitter.com/mennenia)
+- Luc Succes - [l2succes](https://github.com/l2succes) - [@lucsucces](https://twitter.com/lucsucces)
+- Sarah Scott - [sarahscott](https://github.com/sarahscott)
+- David Sheldrick - [ds300](https://github.com/ds300) - [@djsheldrick](https://twitter.com/djsheldrick)
+- Ashley Jelks - [ashleyjelks](https://github.com/ashleyjelks) - [@AshTheYogiDev](https://twitter.com/AshTheYogiDev)
+- Lily Pace - [lilyfromseattle](https://github.com/lilyfromseattle) - [@lilyfromseattle](https://twitter.com/lilyfromseattle)
+- Sarah Weir - [sweir27](https://github.com/sweir27) - [@sweir27](https://twitter.com/sweir27)
+- Yuki Nishijima - [yuki24](https://github.com/yuki24) - [@yuki24](https://twitter.com/yuki24)
+- Jory Steifel - [jorystiefel](https://github.com/jorystiefel)
```
Comment (brainbicycle) [inline docs/thanks.md]: 💯

Comment (ashfurrow) [reply docs/thanks.md]: I totally forgot about this file, really happy to update it 😄

**Comment by ds300** [comment]
awesome work, thanks 🙏

**Pull request merged**